src/parsers/manifest/metaplaylist/metaplaylist_parser.ts

```diff
@@ -293,13 +293,18 @@ function createManifest(
   }
 
   const time = performance.now();
+  const isLastPeriodKnown = !isDynamic ||
+                            mplData.pollInterval === undefined &&
+                            (manifests.length <= 0 ||
+                             manifests[manifests.length - 1].isLastPeriodKnown === true);
```

Comment:
What if `manifests[manifests.length - 1].isLastPeriodKnown` is equal to `undefined`?
Shouldn't it be `undefined`?

We could also remove the possibility for `isLastPeriodKnown` to be undefined considering I don't think it is ever the case

Reply:
done